frontend/PropVal.js

```diff
@@ -149,49 +149,47 @@ function previewProp(val: any, nested: boolean, inverted: boolean, theme: Theme)
 }
 
 function previewArray(val, inverted, theme) {
-  var items = {};
+  var items = [];
   val.slice(0, 3).forEach((item, i) => {
-    items['n' + i] = <PropVal val={item} nested={true} inverted={inverted} theme={theme} />;
-    items['c' + i] = ', ';
+    if (i > 0) {
+      items.push(<span key={i+','}>{', '}</span>);
```

Comment:
Do we ever need a span here? Seems like just text would do.

Reply:
Huh, at first I thought we did– to preserve white space– but I think you're right 😄